### ISSUE TITLE
org `#+call` line returns an error

### ISSUE THREAD

**Issue by brabalan** [hlissner/doom-emacs#634]
### Observed behavior

With the following org file:
``` org
#+name: test
#+BEGIN_SRC emacs-lisp
"foo"
#+END_SRC

#+call: test()
```
when putting the cursor on the last line and doing `C-c C-c` I get a `Wrong type argument: stringp, nil` error.

### Expected behavior

The code block would run and return "foo" (tested with vanilla emacs).

### System information

<details>
<summary>Click to expand</summary>

```
- OS: darwin (x86_64-apple-darwin17.5.0)
- Emacs: 26.1 (May 29, 2018)
- Doom: 2.0.9 (develop 90ddc673371351c4a5057bef268c14803952b498)
- Graphic display: t (daemon: nil)
- System features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS THREADS LCMS2
- Details:
  ```elisp
  uname -a:  Darwin wifi-eduroam-84-146.paris.inria.fr 17.5.0 Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64 x86_64
  modules:   (:feature (evil +everywhere) file-templates (lookup +devdocs +docsets) snippets spellcheck (syntax-checker +childframe) version-control workspaces :completion (ivy +childframe) :ui doom-dashboard doom-modeline doom-quit evil-goggles hl-todo nav-flash neotree (popup +all +defaults) vi-tilde-fringe window-select :emacs ediff electric-indent eshell imenu :tools macos make magit password-store pdf rotate-text tmux upload :lang data emacs-lisp javascript latex ledger markdown ocaml (org +attach +babel +capture +export +present) perl python rest ruby sh web :app :config :private)
  packages:  (ace-link ace-window adaptive-wrap all-the-icons anaconda-mode anzu async auctex auto-compile auto-yasnippet avy bbdb (centered-window :recipe (centered-window :fetcher github :repo anler/centered-window-mode)) cmake-mode coffee-mode command-log-mode company company-anaconda company-auctex company-box company-dict (company-glsl :recipe (company-glsl :fetcher github :repo Kaali/company-glsl)) company-inf-ruby company-irony company-irony-c-headers company-restclient company-shell company-statistics company-web counsel counsel-css counsel-dash counsel-projectile csv-mode cuda-mode demangle-mode (devdocs-lookup :recipe (devdocs-lookup :fetcher github :repo skeeto/devdocs-lookup)) dired-k disaster dockerfile-mode doom-themes dtrt-indent dumb-jump editorconfig elfeed elfeed-org (emacs-snippets :recipe (emacs-snippets :fetcher github :repo hlissner/emacs-snippets :files (*))) emmet-mode eslintd-fix esup evil evil-anzu evil-args evil-collection evil-commentary evil-easymotion evil-ediff evil-embrace evil-escape evil-exchange evil-goggles evil-indent-plus evil-ledger evil-magit evil-matchit evil-mc evil-multiedit evil-numbers evil-org evil-snipe evil-surround evil-textobj-anyblock evil-vimish-fold evil-visualstar (exec-path-from-shell :ignore nil) expand-region flycheck flycheck-cask flycheck-irony flycheck-ledger flycheck-perl6 flycheck-popup-tip flycheck-posframe flyspell flyspell-correct flyspell-correct-ivy git-gutter-fringe git-link git-timemachine gitconfig-mode gitignore-mode glsl-mode graphql-mode haml-mode helpful hide-mode-line highlight-indentation highlight-numbers highlight-quoted hl-todo htmlize hydra imenu-anywhere imenu-list inf-ruby irony irony-eldoc ivy ivy-bibtex ivy-hydra ivy-pass ivy-posframe ivy-rich ivy-rtags ivy-xref js2-mode js2-refactor json-mode ledger-mode less-css-mode macrostep magit magit-gitflow magithub makefile-executor markdown-mode markdown-toc merlin modern-cpp-font-lock nav-flash neotree nodejs-repl nose ob-mongo ob-sql-mode ob-translate opencl-mode (org-bullets :recipe (org-bullets :fetcher github :repo Kaligule/org-bullets)) org-download org-plus-contrib org-tree-slide (osx-clipboard :ignore nil) overseer ox-pandoc ox-reveal pass password-store password-store-otp pcre2el pdf-tools perl6-mode persistent-soft persp-mode pip-requirements projectile pug-mode quelpa quickrun rainbow-delimiters rainbow-mode rake restart-emacs restclient rjsx-mode (rotate-text :recipe (rotate-text :fetcher github :repo debug-ito/rotate-text.el)) rspec-mode rtags ruby-refactor sass-mode shrink-path skewer-mode slim-mode slime smartparens smex solaire-mode ssh-deploy stylus-mode swiper tide toc-org toml-mode tuareg typescript-mode undo-tree use-package vi-tilde-fringe vimrc-mode visual-fill-column web-beautify web-mode wgrep which-key xref-js2 yaml-mode yard-mode yasnippet)
  elc dirs:  n/a
  exec-path: (~/.opam/4.04.0/bin/ ~/bin/ ~/.local/bin/ ~/go/bin/ /usr/local/Cellar/ruby/1.9.3-p194/bin/ /usr/local/share/npm/bin/ ~/.cabal/bin/ /usr/local/bin/ /usr/local/sbin/ /usr/local/bin/ /usr/bin/ /bin/ /usr/sbin/ /sbin/ /Library/TeX/texbin/ /usr/local/MacGPG2/bin/ /opt/X11/bin/ ~/.opam/4.04.0/bin/ ~/bin/ ~/.local/bin/ ~/go/bin/ /usr/local/Cellar/ruby/1.9.3-p194/bin/ /usr/local/share/npm/bin/ ~/.cabal/bin/ /usr/local/sbin/ /usr/local/Cellar/emacs-mac/HEAD-5e088d8/libexec/emacs/26.1/x86_64-apple-darwin17.5.0/)
  ```
</details>

**Comment by hlissner** [comment]
This should be fixed as of 5c8ff3a, let me know if that isn't the case!

**Comment by brabalan** [comment]
It works, thanks!
